### PR TITLE
Pair-feedback

### PR DESCRIPTION
**Pull request by christianbiring1**
- [ ] Give feedback on someone else's project based on clean code best practices.

- [ ] Apply knowledge the student has gained in the past when giving a code review.

- [ ] Perform a simple code review for a coding partner.

- [ ] Understand existing code in a way that allows the student to give a good code review.

### CODE REVIEW

File: src/modules/interactive.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        }
+      }
+
+      taskchecked.style.color = 'black';
+      taskchecked.style.textDecoration = 'none';
+      for (let i = 0; i < tasks.length; i += 1) {
+        if (tasks[i].description === taskchecked.textContent) {
+          tasks[i].completed = false;
+          localStorage.setItem('tasks', JSON.stringify(tasks));
+          console.log(tasks);
```
Comment (ghost) [inline src/modules/interactive.js]: Hey @ christianbiring1, Great Job reaching this point but there's something you still need to work on as I have highlighted above.
- Avoid using console statements to keep your code clean

**Review by ghost — CHANGES_REQUESTED** [review]
##

**Review by ghost — CHANGES_REQUESTED** [review]
##

File: src/modules/interactive.js (hunk 1 above)
Comment (christianbiring1) [reply src/modules/interactive.js]: Yeah yeah! :smile: really.
As I have just mentioned in the issue I just created in the previous lesson, I noticed that this file `interactive.js` was left in my repo while I didn't even use it as you can see. It was only for testing. that's even why I forgot to delete those `console.log`.
let me go ahead and delete it though.

**Review by ghost — APPROVED** [review]
Hey @christianbiring1 
Your project is complete! There is nothing else to say other than... it's time to merge it :
Congratulations! 🥇 🍾 

### Highlight 
-  no linters errors ✔️ 
-   You have followed HTML & CSS best practices✔️ 
-  You have followed JavaScript best practices✔️ 
-  You have followed DRY, KISS, and YAGNI✔️
- Meaningful commit messages


### Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**